### PR TITLE
Update device SDK and toml files

### PR DESCRIPTION
**Pull request by sivakrishnan-intel**
This PR addresses the issue #22.

### CODE REVIEW

File: cmd/res/configuration.toml [hunk 1]
```diff
@@ -50,4 +50,4 @@ FailWaitTime = 10
 [Logging]
 EnableRemote = false
 File = "./device-Modbus.log"
-Level = "DEBUG"
+Level = 'DEBUG'
```
Comment (lenny-intel) [inline cmd/res/configuration.toml]: Also need to restore to INFO

Comment (lenny-intel) [reply cmd/res/configuration.toml]: Required changes seem to have been lost... Should match docker version.

**Comment by cloudxxx8** [comment]
The commits should be squashed if possible

**Comment by lenny-intel** [comment]
@SteveOss, @cloudxxx8 . Something happened with Siva changes, i.e. missing changes to non-docker version. Should we revert the merge or just fix in a new PR?

**Comment by cloudxxx8** [comment]
I noticed that early today and I prefer to open a new PR because revert is also a new PR

**Pull request merged**